include/rocksdb/cache.h

```diff
@@ -127,6 +128,51 @@ extern std::shared_ptr<Cache> NewLRUCache(
 
 extern std::shared_ptr<Cache> NewLRUCache(const LRUCacheOptions& cache_opts);
 
+// Options structure for configuring a SecondaryCache instance based on
+// LRUCache. The LRUCacheOptions.secondary_cache is not used and
+// should not be set.
+struct LRUSecondaryCacheOptions : LRUCacheOptions {
```

Comment:
We typically mark new features as EXPERIMENTAL in the comments. It gives us flexibility to change the APIs in the future.

Reply:
This will be addressed in next commit.